publish_show.py

```diff
@@ -0,0 +1,35 @@
+"""Script to be ran by GH Actions to build the archive and schedule the newsletter"""
+import pprint
```

Comment:
Note for the future: we should get a linter in place to remove unused imports.

Reply:
I'll remove the imports and pre-commit prior to pushing the PR.